lib/puppet/provider/mailalias/augeas.rb

```diff
@@ -19,23 +22,20 @@
 
   def self.get_resource(aug, apath, target)
     malias = {
-      :ensure => :present,
-      :target => target
+      ensure: :present,
+      target: target
     }
-    return nil unless malias[:name] = aug.get("#{apath}/name")
+    return nil unless malias[:name] == aug.get("#{apath}/name")
```

Comment:
This looks like a change, but probably a bugfix. I'd separate this to its own commit.

Reply:
It was also a rubocop offense, why separate it?

Reply:
Because it's a very different thing. The old version is essentially:
```ruby
malias[:name] = aug.get("#{apath}/name")
return nil unless malias[:name]
```

This means `malias[:name]` was always overwritten.

The new version never overwrites `malias[:name]`. If any code depends on that behavior, you're breaking it.

I think a Rubocop compatible change is:
```suggestion
    return nil unless (malias[:name] = aug.get("#{apath}/name"))
```